contrib/linux/actions/service.py

```diff
@@ -70,6 +70,7 @@ def get_linux_distribution():
     re.search(distro, "Redhat")
     or re.search(distro, "Fedora")
     or re.search(distro, "CentOS Linux")
```

Comment:
Not exactly related to this PR, but can we modify this to say `CentOS` instead of `CentOS Linux`? The stream releases won't match the current version, since `platform.linux_distribution()` for stream returns 
```
platform.linux_distribution()
>> ('CentOS Stream', '8', '')
```
and the `lsb_release` way returns
```
→ lsb_release -i -s
CentOSStream
```
On CentOS 7, I guess this will still fail as well with Python >= 3.8
```
→ lsb_release -i -s
CentOS
```

Reply:
I would be happy with that. Could you file a new PR?